scripts/EGAP/files_to_import_structure.py

```diff
@@ -0,0 +1,55 @@
+import os
+import shutil
+import argparse
+from distutils.dir_util import copy_tree
+
+get_item_id = lambda _path: _path.split(os.sep)[-1].split(' ')[0]
```

Comment:
Could you add a comment for what an expected input and output for this would be?